### ISSUE TITLE
Should we rename conversations for action to conversations for agreement?

### ISSUE THREAD

**Issue by bhaugen** [valueflows/intent#2]
I'm connecting to @gcassel 's Agreement-based Organization theme. I think that intents are the bridge between social and economic networks. All the intents I listed in the readme for this repo were about leadups to economic interactions, but you could also have conversations using the same kinds of protocols about forming organizations, or setting organizational rules, or almost any other kind of agreement. 

Can't remember if I posted this issue before, but looked in the usual places and did not see it.

**Comment by bhaugen** [comment]
See also @gcassel 's [Asynchronous Analytical Discussion](https://docs.google.com/document/d/1H55a5TncjaXhyBi9Bf-Uwslce5_FRhOY3BUk5t1rbRg/edit?usp=sharing).

**Comment by gcassel** [comment]
I agree, with an asterisk:   I think that conversations for action is a crucially important subset of conversations for agreement.    -- And one could argue that conversations for action is what really matters to economic networks, but I would argue that that's not as fundamentally sound in developing vocabularies for communities.

I've been thinking lately that we have two basic types of agreements, which can be described roughly as "agreements on opinion/ view"  and "agreements on action".    By this line of thought,  I believe that "agreements on action" are *a subset* of "agreements on opinion".   Here's what I mean:  if we agree to actions, it means that we share an especially decisive opinion: that a plan of action is justified.  We're willing to mutually agree to a plan of action.  That plan of action may be a firm commitment, although it can be more casual.   (I consider commitments to be a subset of the possible relationship-types within plans of action.)

"Agreements on action" could also be called "Intentions", although that's shorthand:  we're really talking about two or more parties mutually agreeing upon the intentions of specific agents or types of agent.

**Comment by bhaugen** [comment]
Agree about subset. Other aspects of the semi-standard Conversations for Action protocol include fulfillment of commitments and evaluation of the whole cycle. Those fit into some conversations, but not others.
